pvlib/solarposition.py

```diff
@@ -1445,3 +1454,96 @@ def sun_rise_set_transit_geometric(times, latitude, longitude, declination,
     sunset = _local_times_from_hours_since_midnight(times, sunset_hour)
     transit = _local_times_from_hours_since_midnight(times, transit_hour)
     return sunrise, sunset, transit
+
+
+def spencer_mc(times, latitude, longitude):
+    """
+    Calculate the solar position using a python implementation of the
+    Spencer (1972) formulation provided by meteocontrol
```

Comment:
My preference is a code comment saying who added the code and what their affiliation is. I don't think this belongs in rendered documentation.

Reply:
Some subtle form of recognition seems fine with me.  But if the method is identical to spencer's paper, then the suffix '_mc' might be too much.